will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml

```diff
@@ -0,0 +1,67 @@
+mappings:
+  - name: mpingram_etl
```

Comment:
Might want to change this, any ETL jobs will override his indices. If you want to rely on his indices and not run ETL until you get the hang of it just remove this file, or else change the name.

Reply:
Thanks! I'll update to my name